saleor/checkout/views/__init__.py

```diff
@@ -218,8 +218,11 @@ def update_checkout_line(request, checkout, variant_id):
     if form.is_valid():
         form.save()
         checkout.refresh_from_db()
-        checkout_line.refresh_from_db()
-        subtotal = get_display_price(checkout_line.get_total(discounts, taxes))
+        checkout_line = checkout.lines.filter(variant_id=variant_id).first()
```

Comment:
I would add a comment explain why we need this. If I understand correctly, it's because if the quantity is set to 0, it will delete the line.

Reply:
Exactly